src/energyplus/Test/GroundHeatExchangerVertical_GTest.cpp

```diff
@@ -0,0 +1,121 @@
+/***********************************************************************************************************************
+*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+*  See also https://openstudio.net/license
+***********************************************************************************************************************/
+
+#include <gtest/gtest.h>
+#include "EnergyPlusFixture.hpp"
+
+#include "../ForwardTranslator.hpp"
+#include "../ReverseTranslator.hpp"
+
+#include "../../model/Model.hpp"
+#include "../../model/GroundHeatExchangerVertical.hpp"
+#include "../../model/GroundHeatExchangerVertical_Impl.hpp"
+#include "../../model/PlantLoop.hpp"
+#include "../../model/Node.hpp"
+#include "../../model/Node_Impl.hpp"
+
+#include "../../utilities/idf/IdfFile.hpp"
+#include "../../utilities/idf/Workspace.hpp"
+#include "../../utilities/idf/IdfObject.hpp"
+#include "../../utilities/idf/WorkspaceObject.hpp"
+
+#include <utilities/idd/GroundHeatExchanger_System_FieldEnums.hxx>
+#include <utilities/idd/Site_GroundTemperature_Undisturbed_KusudaAchenbach_FieldEnums.hxx>
+#include <utilities/idd/GroundHeatExchanger_ResponseFactors_FieldEnums.hxx>
+#include <utilities/idd/GroundHeatExchanger_Vertical_Properties_FieldEnums.hxx>
+#include <utilities/idd/IddEnums.hxx>
+
+using namespace openstudio::energyplus;
+using namespace openstudio::model;
+using namespace openstudio;
+
+TEST_F(EnergyPlusFixture, ForwardTranslator_GroundHeatExchangerVertical) {
+
+  ForwardTranslator ft;
+
+  // Create a model
+  Model m;
+
+  GroundHeatExchangerVertical ghx(m);
+
+  EXPECT_TRUE(ghx.setDesignFlowRate(0.004));
+  EXPECT_TRUE(ghx.setNumberofBoreHoles(100));
+  EXPECT_TRUE(ghx.setBoreHoleLength(80.0));
+  EXPECT_TRUE(ghx.setBoreHoleRadius(0.7E-01));
+  EXPECT_TRUE(ghx.setGroundThermalConductivity(0.7));
+  EXPECT_TRUE(ghx.setGroundThermalHeatCapacity(0.3E+07));
```

Comment:
Like I mentioned. Do these methods do anything now?

Reply:
These 2 still translate to the ghxVert object (see explanation above).